docs/publish/publish-add-in-vs-code.md

```diff
@@ -43,9 +44,9 @@ When your add-in is working as desired, you can publish it directly through Visu
 
 10. To deploy, select the Files explorer, select and hold (right-click) your **dist** folder, and choose **Deploy to Static Website via Azure Storage**. When prompted, select the storage account you created previously.
```

Comment:
```suggestion
10. To deploy, select File Explorer, select and hold (right-click) your **dist** folder, and choose **Deploy to Static Website via Azure Storage**. When prompted, select the storage account you created previously.
```

Reply:
Per [VS Code docs](https://code.visualstudio.com/docs/getstarted/userinterface). If you accept this suggestion, line 110 will also need to be updated (I couldn't add a suggestion to the line).